layouts/partials/docs/hero.html

```diff
@@ -73,7 +73,8 @@
               </span>
               {{ end }}
             </a>
-            {{ else if ((ne "/v2.3/" $new_version) | and (ne "/v3.1/" $new_version)) }}
+            {{/* check if we have a old-style long version number */}}
+            {{ else if gt (strings.RuneCount $new_version) 6 }}
```

Comment:
By the time we're done with issue https://github.com/etcd-io/website/issues/152, this `else if` will be removed. I know this is a kludge, but hopefully it'll only be here for a day or two.